Marlin/src/lcd/extui/lib/mks_ui/draw_print_file.cpp

```diff
@@ -243,7 +243,7 @@ void lv_draw_print_file(void) {
   }
   */
 }
-static char test_public_buf_l[40];
+static char test_public_buf_l[FILE_NUM][SHORT_NEME_LEN + 6];
```

Comment:
Is `SHORT_NEME` supposed to be `SHORT_NAME`?

Reply:
And why `+ 6`?

Better to used a named constant for a "magic number", so the reason can be seen.

Reply:
to hold SHORT_NEME_LEN + "F:/" ... maybe a few bytes more.